themes/default/layouts/partials/banner.html

```diff
@@ -1,6 +1,6 @@
-{{ $summary := "has launched. Program the cloud like never before! 🚀" }}
-{{ $url := relref . "/blog/pulumi-3-0" }}
-{{ $label := "Pulumi 3.0"}}
+{{ $summary := "Watch the Pulumi 3.0 annoucements video." }}
+{{ $url := relref . "/pulumi-up" }}
+{{ $label := "Your cloud. Your language. Your way."}}
```

Comment:
This would work better IMO if "Watch the announcements" were linked, rather than "your cloud", since the action is to watch the announcements.

Two other thoughts:

* Would "Watch the talks" or "Watch the sessions" or something similar maybe work better, since there's more than just announcements there? I just paused on the word announcements because it made me wonder whether it'd just be the intro talk or whether all of them would be there. Something to think about maybe.

* I think @susanev mentioned replacing the rocket emoji with a new one -- any ideas, or just want to leave it out?

Reply:
👍

Reply:
I'm not really sure what emoji to use so open to suggestions. For now I've left it blank but if y'all think of good, feel free to add it in!